### ISSUE TITLE
Why is lsp-tailwindcss--company-dash-hack neccasary? What would a non-hacky sollution entail?

### ISSUE THREAD

**Issue by noah-evans** [merrickluo/lsp-tailwindcss#63]
It works but it makes company more irritating to use since the completion window jumps when you type a dash.

What is preventing us from typing dashes in completions without making them trigger characters?

**Comment by merrickluo** [comment]
yeah, it's a workaround, it seems lsp-mode treats it as a word separator, if it's not set as a trigger character, it won't send the autoComplete request to the server hence we can't get auto-completion when input -. 

let me try to see if this has improved since then.
